backend/coreapp/middleware.py

```diff
@@ -51,6 +51,7 @@ def middleware(request: Request) -> Response:
             "node-fetch" in request.headers["User-Agent"]
```

Comment:
Assuming user agent is a string, can this check be removed as it'll be caught by `"node" in ...`?

Reply:
I thought it wasn't a string, but yeah you're right